pkg/scaffold/util/util.go

```diff
@@ -0,0 +1,91 @@
+// Copyright 2018 The Operator-SDK Authors
```

Comment:
rename this to `fileutil.go` to be more specific?

Reply:
Renamed to `file_util.go` and moved to `pkg/util` directory.